### DISCUSSION TITLE
Smart Meter only - Sensors not showing up

### DISCUSSION THREAD

**Discussion by Tifuso** [thomluther/ha-anker-solix#123] [Q&A] [answered]
Hi everyone,

I just installed the Anker smart meter without any other Anker Solix device.

Firmware of the Smart Meter: v0.0.0.29, Anker 2.1.1, HACS 2.0.1

My problem is, that i can only see these 5 entities:
- binary_sensor.smart_meter_wifi_verbindung
- button.smart_meter_details_aktualisieren
- sensor.smart_meter_bws_uberhang
- sensor.smart_meter_firmware
- sensor.smart_meter_etikett

But I am missing the Grid Sensors? Can I add them somehow? Or would I need the whole Solix system?

Sorry if this question is nooby or the answer is completely obvious.

Best regards!

**Comment by thomluther** [comment, marked as answer]
You need to define a power system in the Anker App and add devices to the system. Standalone devices do not provide any values via the cloud api, only systems do. To create a system, you need a major system device such as the solarbank.
